portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -79,42 +75,35 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(json);
   }
 
-    private String getUsername(String id) {
-        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-        Query query =
-            new Query("user")
-                .setFilter(new Query.FilterPredicate("id", Query.FilterOperator.EQUAL, id));
-        PreparedQuery results = datastore.prepare(query);
-        Entity entity = results.asSingleEntity();
-
-        if (entity == null) {
-            return null;
-        }
-
-        System.out.println("======NOT+NULLLLLLL=======");
+  private String getUsername(String id) {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query = new Query("user").setFilter(
```

Comment:
Be consistent with naming conventions.  If your convention is that entities start with capital letters and parameters / fields start with lowercase (like you've done with "Comment" and "comment"), then you should also capitalize "User".

Reply:
Made user a capital letter